components/shared/net/lib.rs

```diff
@@ -328,6 +328,15 @@ impl FetchTaskTarget for IpcSender<FetchResponseMsg> {
     }
 }
 
+impl FetchTaskTarget for IpcSender<WebSocketNetworkEvent> {
+    fn process_request_body(&mut self, _: &Request) {}
+    fn process_request_eof(&mut self, _: &Request) {}
+    fn process_response(&mut self, _: &Request, _: &Response) {}
+    fn process_response_chunk(&mut self, _: &Request, _: Vec<u8>) {}
+    fn process_response_eof(&mut self, _: &Request, _: &Response) {}
+    fn process_csp_violations(&mut self, _: &Request, _: Vec<csp::Violation>) {}
```

Comment:
We will need to add a real implementation to process_csp_violations that sends the ReportCspViolatiobs message from https://doc.servo.org/net_traits/enum.WebSocketNetworkEvent.html